group_boards.go

```diff
@@ -136,6 +136,66 @@ func (s *GroupIssueBoardsService) GetGroupIssueBoard(gid interface{}, board int,
 	return gib, resp, err
 }
 
+type UpdateGroupIssueBoardOptions struct {
+	ID          *string `url:"id" json:"id"`
+	BoardID     *int    `url:"board_id" json:"board_id"`
+	Name        *string `url:"name" json:"name"`
+	AssigneeID  *int    `url:"assignee_id" json:"assignee_id"`
+	MilestoneID *int    `url:"milestone_id" json:"milestone_id"`
+	Labels      *string `url:"labels" json:"labels"`
```

Comment:
There is a `Label` type in this package which is probably a better fit for this one. Of do you have a specific reason to use a `*string` here?

Reply:
And these fields all need `omitempty`...

Reply:
i'm using `*string` because the docs say that the `labels` are "Comma-separated list of label names which the board should be scoped to". Because of this i think it should be a string.

Reply:
I know, but that is what I created the `Label` type for. It has custom (un)marshaling methods so it can be used as a slice but will be serialized as a comma separated string. See here for more info: https://github.com/xanzy/go-gitlab/blob/master/issues.go#L105

Reply:
if we have the `Labels` type then it's better to use it. i changed it from string to Labels now.